skynet/nodeos.py

```diff
@@ -88,6 +88,8 @@ def open_nodeos(cleanup: bool = True):
         logging.info(f'GPU KEYS: {(priv, pub)}')
         cleos.new_account('telos.gpu', ram=4200000, key=pub)
 
+        # lol, magic #s much XD ?
+        # TODO, why is there 4 workers/keys?
```

Comment:
Just seeding accounts for tests, this whole module should be updated to latest py-leap apis, but thats for when we fix the test suite.